debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PgOid.java

```diff
@@ -38,6 +40,9 @@ protected static int jdbcColumnToOid(Column column) {
             return valueOf(typeName);
         } catch (PSQLException e) {
             // not known by the driver PG driver
+            if (typeName.toUpperCase().equals("TSTZRANGE")) {
```

Comment:
Why is this done in the catch block? Couldn't you check this one before calling `valueOf()` above?